src/ralph/deployment/models.py

```diff
@@ -87,7 +86,7 @@ def autocomplete_str(self):
 <br>  - ralph_instance (eg. '{ralph_instance}')
 <br>  - service_env (eg. 'Backup systems - prod')
 <br>  - service_uid (eg. 'sc-123')
-""".format(ralph_instance=settings.RALPH_INSTANCE.rstrip('/')).strip()  # noqa
+""".format(ralph_instance='http://ralph.example.com').strip()  # noqa
```

Comment:
you're fixing the effect, not the cause. Can't you do sth like [that](https://stackoverflow.com/a/29233569/761782) or move generating this help text to form field (not model field)? In my opinion we should provide proper ralph url in this help text, not example one.

Reply:
Sounds sensible

Reply:
Done